gulpfile.js

```diff
@@ -249,9 +226,9 @@ function changedAssets(prevManifest, newManifest) {
 
 var debouncedLiveReload = debounce(function () {
```

Comment:
The live reload server is used for testing the sidebar, so presumably won't be needed in the main repository any more assuming that the gulp process watching/building client assets is run from the other repo.